nixos/modules/installer/cd-dvd/installation-cd-graphical-calamares-gnome.nix

```diff
@@ -0,0 +1,59 @@
+# This module defines a NixOS installation CD that contains GNOME.
+
+{ pkgs, ... }:
+
+{
+  imports = [ ./installation-cd-graphical-calamares.nix ];
+
+  isoImage.edition = "gnome";
+
+  services.xserver.desktopManager.gnome = {
+    # Add Firefox and other tools useful for installation to the launcher
+    favoriteAppsOverride = ''
+      [org.gnome.shell]
+      favorite-apps=[ 'firefox.desktop', 'nixos-manual.desktop', 'org.gnome.Console.desktop', 'org.gnome.Nautilus.desktop', 'gparted.desktop', 'io.calamares.calamares.desktop' ]
+    '';
+
+    # Override GNOME defaults to disable GNOME tour and disable suspend
+    extraGSettingsOverrides = ''
+      [org.gnome.shell]
+      welcome-dialog-last-shown-version='9999999999'
+
+      [org.gnome.settings-daemon.plugins.power]
+      sleep-inactive-ac-type='nothing'
+      sleep-inactive-battery-type='nothing'
+    '';
+
+    extraGSettingsOverridePackages = [ pkgs.gnome.gnome-settings-daemon ];
+
+    enable = true;
+  };
+
+  # Theme calamares with GNOME theme
+  qt5 = {
+    enable = true;
+    platformTheme = "gnome";
+  };
+
+  # Fix scaling for calamares
+  environment.sessionVariables = {
+    QT_QPA_PLATFORM = "wayland";
```

Comment:
Will this work if someone boots into xorg session, e.g. on unsupported hardware? (You can simulate using `services.xserver.displayManager.defaultSession = "gnome-xorg";`)

Reply:
You're right, that's a bad idea. Testing if setting to `QT_QPA_PLATFORM = "$([[ $XDG_SESSION_TYPE = \"wayland\" ]] && echo \"wayland\")";` will work. Although a better solution might be introducing this patch from Fedora into `qtbase`: https://src.fedoraproject.org/rpms/qt5-qtbase/raw/main/f/qtbase-use-wayland-on-gnome.patch

Reply:
`environment.sessionVariables` uses PAM so you cannot use shell there.

Reply:
Looks like the code has been [removed in 6.3/6.2](https://github.com/qt/qtbase/commit/dda7dab8274991e4a61a97c352d4367f8f815bb9). Not sure how usable it is in Qt 5.

Reply:
Seems like removed in QT5 here: https://codereview.qt-project.org/c/qt/qtbase/+/231227. And it works with `environment.sessionVariables`, but probably would be better to use `environment.variables` right?

Reply:
Ok, everything seems to work fine with `environment.variables`. Pushed changes and uploading new GNOME ISO